src/test/scala/org/broadinstitute/dsde/firecloud/utils/TSVFormatterSpec.scala

```diff
@@ -144,7 +168,10 @@ class TSVFormatterSpec extends FreeSpec with ScalaFutures with Matchers with Ins
     val headers = lines.head.split("\t")
 
     // Make sure that all of the post-rename values exist in the list of headers from the TSV file
-    forAll (headerRenamingMap.values.toList) { x => headers should contain(x) }
+    // for all non-set types
```

Comment:
I'm wondering if we need to also test that each line has the same number of values as columns, given this fix deals with sparse data?